crates/floresta-wire/src/p2p_wire/chain_selector.rs

```diff
@@ -235,16 +235,10 @@ where
     ) -> Result<Option<PeerId>, WireError> {
         let (mut height, mut hash) = self.chain.get_best_block()?;
         let mut prev_height = 0;
-        let agree = false;
         // we first norrow down the possible fork point to a couple of blocks, looking
         // for all blocks in a linear search would be too slow
         loop {
             // ask both peers for the utreexo state
-            self.send_to_peer(peer1, NodeRequest::GetUtreexoState((hash, height)))
-                .await?;
-            self.send_to_peer(peer2, NodeRequest::GetUtreexoState((hash, height)))
-                .await?;
-
```

Comment:
Removed these two `GetUtreexoState` requests because there is not method implemented to receive the response in this funciton. Also these same requests are sent once again in the `grab_both_peers_version` function.